b2sdk/b2http.py

```diff
@@ -413,6 +441,8 @@ def _translate_and_retry(cls, fcn, try_count, post_params=None):
         """
         # For all but the last try, catch the exception.
         wait_time = 1.0
+        max_wait_time = 64
+        rng = Random()
```

Comment:
any particular reason for having a separately seeded generator for each call? The execution time of initialisation is humungous compared to a single number generation:

```
22:46:39 reef@~
$ python3 -m timeit 'import random; random.Random()'
10000 loops, best of 5: 25.1 usec per loop
22:47:01 reef@~
$ python3 -m timeit 'import random; random.Random().random()'
10000 loops, best of 5: 25.6 usec per loop
22:47:10 reef@~
$ python3 -m timeit 'import random; random.random()'
2000000 loops, best of 5: 123 nsec per loop
```

but completely negligible compared to HTTP IO. I'm just curious as to why.

Reply:
you are right, I didn't know that's so slow!